docs/settings.rst

```diff
@@ -785,6 +785,19 @@ the server will return NODATA for A/AAAA queries for such names.
   In PowerDNS Authoritative Server 4.0.x, this setting did not exist and
   ALIAS was always expanded.
 
+.. _setting-resolve-across-domains:
+
+``resolve-across-domains``
+--------------------------
+
+-  Boolean
+-  Default: yes
+
+If this is enabled, CNAME records and other referrals will be resolved as long as their targets exist in any local backend.
+Can be disabled to allow for different authorities managing zones in the same server instance.
+
+Regardless of this setting, external targets are never resolved.
```

Comment:
I hope this line won't confuse `ALIAS` users

Reply:
maybe we should list all "other referrals" to be clear.

Reply:
I had to re-read my own line 5 times now to understand what I meant. Hard to come up with something descriptive though.

Maybe just:

> Referrals not available in local backends are never resolved.
> SVCB referrals are never resolved across domains.
> ALIAS is not impacted by this setting.

Reply:
s/domains/zones/, love it otherwise

Reply:
> love it otherwise

done

> s/domains/zones/

should I also do that to the setting *name*? various other settings use "domain" though

Reply:
done!